sections/placeholders.md

```diff
@@ -0,0 +1,117 @@
+# Placeholder Resources
+
+##### Endpoint: `/api/v1/placeholder_resources`
+
+Placeholder resources can be used for temporary resourcing on projects where the final resources are not yet known. Placeholders behave quite similarly to users, with a few differences discussed below.
+
+## List Placeholder Resources (index)
+
+##### Optional Parameters:
+
+| **Parameter** | **Description** |
+| ------------- | --------------- |
+| fields | A comma separated list of additional fields to include in the response [ "assignments", "custom_field_values"] |
```

Comment:
Is this true? If so, both `assignments` and `custom_field_values` are serialized. Do those whole objects come back if you add that key? _All_ assignments?